conbench/units.py

```diff
@@ -42,11 +45,17 @@ def fmt_unit(value: Optional[float], unit) -> Optional[str]:
     if value is None:
         return None
 
-    if value == math.nan:
+    if value is math.nan:
```

Comment:
It seems to _not_ be a question of identity :)

> To check whether a number is a NaN, use the [isnan()](https://docs.python.org/3/library/math.html#math.isnan) function to test for NaNs instead of `is` or `==`

Gotcha.